### ISSUE TITLE
Check indentation of method arguments spanning multiple lines

### ISSUE THREAD

**Issue by bquorning** [bbatsov/rubocop#1144]
Rubocop with default configuration detects no offenses on this code:
```rb
def run(first, second = {})
  puts "#{first} #{second}"
end

def defaults
  { a: 1 }
end

run(:foo, defaults.merge(
   bar: 3
  )
 )

run(
   :foo,
   bar: 3
 )
```
I do, however. All of the arguments to the `#run` calls are mis-indented.

**Comment by bquorning** [comment]
Great work on the `FirstParameterIndentation` cop.

Would it be possible to check the ending parentheses, too? RuboCop still reports no offenses for this code:

```ruby
def foo
  puts('x'
                            )
end

def bar
  puts(
    'y'
                            )
end
```

**Comment by bbatsov** [comment]
It's possible, but it should be done by a dedicated cop IMO.

On 23 February 2015 at 14:57, Benjamin Quorning <notifications@github.com>
wrote:

> Great work on the FirstParameterIndentation cop.
>
> Would it be possible to check the ending parentheses, too? RuboCop still
> reports no offenses for this code:
>
> def foo
>   puts('x'
>                             )end
> def bar
>   puts(
>     'y'
>                             )end
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bbatsov/rubocop/issues/1144#issuecomment-75536453>.
>



-- 
Best Regards,
Bozhidar Batsov

http://www.batsov.com
